Week2/README.md

```diff
@@ -178,34 +191,34 @@ To learn more about this topic, check out the following:
 Let’s say we wanted to get a list of those students and the details of their teacher. This would be a perfect fit for an inner join, since an inner join returns records at the intersection of the two tables.
 
 ```sql
-SELECT s.first_name, s.last_name, s.gender, s.grade, t.full_name
-FROM students s
-INNER JOIN teachers t
-ON s.teacher_number = t.teacher_number
+SELECT students.first_name, students.last_name, students.gender, students.grade, teachers.full_name
+FROM students
+INNER JOIN teachers
+ON students.teacher_number = teachers.teacher_number
 ```
 
-#### Self Joins
-
-A self join is a join in which a table is joined with itself (which is also called Unary relationships), especially when the table has a FOREIGN KEY which references its own PRIMARY KEY. To join a table itself means that each row of the table is combined with itself and with every other row of the table. In department, we want to get the employee's information with their direct manager. Here, each row in employees has a foreign key to itself as manager_id:
+When you join two tables there can be columns in both tables with the same name.
```

Comment:
IMHO, This description is for joins in general. Not for self joins.
Perhaps, go through the example of self-join again ?

Reply:
I removed the whole self join part and added this new line to the text about joins. 
So you are absolutely correct ;)
I have never used `SELECT * FROM table1, table2` ever and didn't find sources that advised this kind of SQL.
And even it where correct SQL it seems like a really unique thing only to be used in rare circumstances.

Reply:
I agree that it is rare but sometimes it is the only solution for the problem.
And we do cover all other types of joins, so I would add this for completeness.

[Example of self join ](https://www.w3schools.com/sql/sql_join_self.asp) on w3schools is one example of a self-join.
Another example is `finding employees and their reporting managers who are also employees`.

Explanation of joins is good and we can keep it but I think it helps to give students a perspective on self join.

Reply:
One of the things I want to start doing more often is highlight the parts that students will be using a lot and should focus on and add things that we feel they should know about but won't be using too often to an 'extra reading' part. This is because I feel both of you are right, students get a huge amount of info dumped on them so you want to limit it to things they will need during their first year of their jobs, but you also want them to get the full picture.

So information like this where this type of self join is apparently not used a lot would be great to give them. That way they can really practice the most used cases and keep this information in the back of their mind.

Can we do something like that maybe?

Reply:
I've added a note about extra curriculum content on line 235.

And the w3schools example of self joins is strange to me. I would rather do 
`SELECT * FROM employees JOIN employees AS managers ON managers.id = employees.manager_id`
compared to 
`SELECT * FROM employees, employees AS managers WHERE managers.id = employees.id` 

I would like to teach students to `JOIN` using `ON` and never use `WHERE` statements to join.
So teaching them this style of self joins seems counter productive.

Reply:
I agree with you on stressing the use of `JOIN` keyword.